meson.build

```diff
@@ -103,12 +103,16 @@ configure_file(
   install_dir: dbus_dir
 )
 
-# Nautilus Extension
+# Nautilus/Nemo Extension
```

Comment:
I think if we're installing the file we should just add a new switch for nemo. Meson is mostly for packagers and they're likely to want one or the other.

Reply:
Fair enough, I'll do that.

Reply:
Any opinion on the filename? I was perfectly happy just leaving it as `nautilus-gsconnect.py`, since it's literally the exact same file... but I could also see renaming that one file to `gsconnect-plugin.py` or `filemanager-gsconnect.py` or something similarly generic, to install in both locations.

Reply:
I think it's fine as-is for now. The meson version we "depend on" is quite old; at some point it will be worth updating the whole build system and there's options line `rename` when installing.